### PR TITLE
Backchannel logout functionality added to sample playground2 app

### PR DESCRIPTION
**Pull request by ashendes**

### CODE REVIEW

File: modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/OIDCBackchannelLogoutServlet.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package org.wso2.sample.identity.oauth2;
```
Comment (darshanasbg) [inline modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/OIDCBackchannelLogoutServlet.java]: Add license

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-get-access-token.jsp [hunk 2]
```diff
@@ -51,6 +53,10 @@
         String idToken = oAuthResponse.getParam("id_token");
         if (idToken != null) {
             session.setAttribute("id_token", idToken);
+            String sid = SessionIdStore.getSid(idToken);
+            if(sid!=null) {
```
Comment (darshanasbg) [inline modules/samples/oauth2/playground2/src/main/webapp/oauth2-get-access-token.jsp]: fix formatting

File: modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/OIDCBackchannelLogoutServlet.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            log.info("Logout token: " + req.getParameter("logoutToken"));
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        HttpSession session = SessionIdStore.getSession(sid);
+        log.info("Session invalidated.");
+        session.invalidate();
+
+    }
+}
```
Comment (darshanasbg) [inline modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/OIDCBackchannelLogoutServlet.java]: Add a new line

File: modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/OIDCBackchannelLogoutServlet.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException,
+            IOException {
+        log.info("Logout request received.");
+
+        String sid = null;
+        try {
+            sid = (String) SignedJWT.parse(req.getParameter("logoutToken")).getJWTClaimsSet().getClaim("sid");
+            log.info("Logout token: " + req.getParameter("logoutToken"));
+        } catch (ParseException e) {
+            e.printStackTrace();
```
Comment (darshanasbg) [inline modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/OIDCBackchannelLogoutServlet.java]: use log.error()

File: modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/SessionIdStore.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
+package org.wso2.sample.identity.oauth2;
```
Comment (denuwanthi) [inline modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/SessionIdStore.java]: add license headers

Comment (denuwanthi) [inline modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/SessionIdStore.java]: add license headers

Comment (denuwanthi) [inline modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/SessionIdStore.java]: Add licnese

**Comment by darshanasbg** [comment]
@ashendes Please send a new PR with mentioned fixes

**Pull request closed without merging**